website/docs/cloud-docs/vcs/troubleshooting.mdx

```diff
@@ -82,6 +82,16 @@ Note that the "SSH Key" section in a workspace's settings is only used for mid-r
 
 ## General
 
+### Failure to retrieve configuration version
+
+If Terraform Cloud displays an error about configuration version retrieval, the VCS provider is likely not responding to Terraform Cloud's requests. 
+
+```
+Error: failed to retrieve configuration
+Retrieving the configuration version failed with the error: Timed out while fetching the configuration version..
+```
+This can happen if there is a network problem between Terraform Cloud and the VCS provider. Push a new commit with any content to a branch in the problem repository to trigger a new configuration version. New commits trigger the repository's associated webhook, creating a new run and configuration version in Terraform Cloud.
```

Comment:
```suggestion
This can happen if there is a network problem between Terraform Cloud and the VCS provider. Push a new commit to a branch in the problem repository to trigger a new configuration version. New commits trigger the repository's associated webhook, creating a new run and configuration version in Terraform Cloud.
```

Reply:
does it have to be a specific branch that's associated with the workspace, or any branch? i think this might depend on the workspace settings, but maybe good to double check?